contracts/CO30_Rob_BomuGaru.swu_public_salman_map/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,12 @@
+<font color='#E3D310'>PRIMARY</font>
+- Locate and remove all explosive ordanance within the AO
+
+<font color='#595305'>SECONDARY</font>
+- Neutralize all ISIS forces within and around Al Salman
+
+
+
+<font color='#E3D310'>RESTRICTIONS</font>
+- Dispose of the mines without blowing them up. It is suspected they have been placed on densily populated routes
```

Comment:
```suggestion
- Dispose of the mines without blowing them up on densely populated routes
```
It is standard for our EODs to do controlled detonation when possible, as it is the safest. They should already avoid doing so near infrastructure, but this is still a good reminder